torch/csrc/distributed/rpc/process_group_agent.cpp

```diff
@@ -456,10 +466,22 @@ void ProcessGroupAgent::enqueueRecv(RecvWork work) {
 }
 
 void ProcessGroupAgent::listenLoop() {
-  while (true) {
+  while (rpcRunning_.load()) {
```

Comment:
Why do we need to check `rpcRunning_.load()` twice in each iteration?  Will it cause any problem if we do `while(true)` here? Or is it just to save the overhead of creating the preamble?